src/main/java/org/schemaspy/output/dot/schemaspy/graph/EmptyGraph.java

```diff
@@ -0,0 +1,13 @@
+package org.schemaspy.output.dot.schemaspy.graph;
+
+public class EmptyGraph implements Graph {
+
+    public static final EmptyGraph EMPTY = new EmptyGraph();
```

Comment:
Remove and make constructor public

Reply:
As earlier comment, this is caused by doing late evaluation in the object graph. This class shouldn't exist.